link/src/main/java/com/stripe/android/link/injection/LinkCommonModule.kt

```diff
@@ -35,7 +35,7 @@ internal interface LinkCommonModule {
             @IOContext workContext: CoroutineContext,
         ): ConsumersApiService = ConsumersApiServiceImpl(
             appInfo = Stripe.appInfo,
-            sdkVersion = ApiVersion(betas = emptySet()).code,
+            sdkVersion = StripeSdkVersion.VERSION,
```

Comment:
This would be an easy thing to add a test for with `NetworkRule`!

Reply:
yes! I was waiting for the PR to merge, now it is! on it.

Reply:
https://github.com/stripe/stripe-android/pull/6154/commits/ecbb81f31560ff86e7c0edd7bf66e62c5588621f